projects/packages/forms/src/blocks/shared/hooks/use-form-style-outline-classes-and-styles.js

```diff
@@ -131,17 +134,8 @@ export default function useFormStyleOutlineClassesAndStyles( {
 			};
 		}
 		if ( formStyle === FORM_STYLE.ANIMATED ) {
-			const borderLeftSize =
-				getIntAsPxValue( blockBorderClassesAndStyles?.style?.borderWidth ) ||
-				blockBorderClassesAndStyles?.style?.borderLeftWidth ||
-				globalBorderClassesAndStyles?.style?.borderWidth ||
-				globalBorderClassesAndStyles?.style?.borderLeftWidth ||
-				'1px';
 			styleSpecificCssVars = {
-				'--jetpack--contact-form--left-offset': `calc(var(--jetpack--contact-form--input-padding-left, 16px) + ${ borderLeftSize })`,
-				'--jetpack--contact-form--label-left':
-					'max(var(--jetpack--contact-form--left-offset), var(--jetpack--contact-form--border-radius))',
-				'--jetpack--contact-form--field-padding': `calc(var(--jetpack--contact-form--label-left) - ${ borderLeftSize })`,
+				'--jetpack--contact-form--animated-left-offset': borderRadius ? '32px' : '16px',
```

Comment:
The `borderRadius` might also be `'0px'` as well as a falsey, so we can end up with an inconsistent behavior.